services/minespace-web/webpack.config.ts

```diff
@@ -215,7 +214,7 @@ const prodConfig = merge([
   }),
   parts.bundleOptimization({
     options: {
-      maxSize: 3000000,
+      // maxSize: 3000000,
```

Comment:
Note for the future: the bundle splitting was causing errors in loading, follow up ticket made.